Tools/cases_generator/tier1_tail_call_generator.py

```diff
@@ -0,0 +1,160 @@
+"""Generate the main interpreter tail call handlers.
+Reads the instruction definitions from bytecodes.c.
+Writes the cases to generated_tail_call_handlers.c.h, which is #included in ceval.c.
```

Comment:
Marking this file as generated in `.gitattributes` would be good, otherwise github would render it in PR reviews. 

https://github.com/python/cpython/blob/1b39b502d33c68f52fd775c4e6c2174baddd40bd/.gitattributes#L61-L68